src/filter.ts

```diff
@@ -433,6 +433,10 @@ type Combine<U, K extends string> = U extends unknown
     ? U & Partial<Record<Exclude<K, keyof U>, undefined>>
     : never;
 
+export type FilterE<Q extends FilterQuery> = PerformQueryE<
```

Comment:
This is a great PQ, I'd just love to have a better suffix than `E`. What does it stand for, extension maybe?

Reply:
Yes. I agree :D

Reply:
Let me throw a bunch of random names at you:
- FilterExtension
- FilterCore
- FilterAddition
- FilterCasing
- FilterShell
- FilterCover
- FilterSide
- FilterWing
- FilterAugmentation
- FilterEnhancement
- FilterAppendix
- FilterAttachment
- FilterAddendum

I think I prefer `FilterCore`. Which one do you like best?

Reply:
`FilterCore`, too

Reply:
Alright, let's rename then. Can you take care of this?

Reply:
What about exports and typedoc comments?

Reply:
I don't really think we need to export the `*Core` types. I see no reason to do this, as userland code would likely always want to use the full types. If there's ever demand for this, we can still export it in the future.

JSDoc comments are only necessary for exported stuff. Internal stuff may or may not have it, I'm mostly only doing this when things are hard to understand without comments.

Reply:
Great. Then it's done